applications/MeshingApplication/custom_utilities/mmg/mmg_utilities.cpp

```diff
@@ -2469,7 +2469,7 @@ void MmgUtilities<TMMGLibrary>::OutputReferenceEntitities(
     std::filebuf cond_buffer;
     cond_buffer.open(rOutputName + ".cond.ref.json",std::ios::out);
     std::ostream cond_os(&cond_buffer);
-    cond_os << r_cond_json_text;
+    cond_os << r_cond_json_text << std::flush;
```

Comment:
@roigcarlo I added this to avoid problems in  the tests, caused from files not being written totally before reading them, is this OK?

Reply:
:+1: